core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/components/dot-category-field-search/dot-category-field-search.component.ts

```diff
@@ -0,0 +1,85 @@
+import { CommonModule } from '@angular/common';
+import {
+    AfterViewInit,
+    ChangeDetectionStrategy,
+    Component,
+    DestroyRef,
+    EventEmitter,
+    inject,
+    input,
+    Output
+} from '@angular/core';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
+import { FormControl, ReactiveFormsModule } from '@angular/forms';
+
+import { InputTextModule } from 'primeng/inputtext';
+
+import { debounceTime, filter } from 'rxjs/operators';
+
+import { DotMessagePipe } from '@dotcms/ui';
+
+export const DEBOUNCE_TIME = 300;
+
+const MINIMUM_CHARACTERS = 3;
+
+@Component({
+    selector: 'dot-category-field-search',
+    standalone: true,
+    imports: [CommonModule, DotMessagePipe, InputTextModule, ReactiveFormsModule],
+    templateUrl: './dot-category-field-search.component.html',
+    styleUrl: './dot-category-field-search.component.scss',
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class DotCategoryFieldSearchComponent implements AfterViewInit {
+    searchControl = new FormControl();
+
+    /**
+     * Represent a EventEmitter for the term the user want to filter
+     *
+     */
+    @Output() term = new EventEmitter<string>();
+
+    /**
+     * Represent a EventEmitter to notify we want change the mode to `list`.
+     */
+    @Output() changeMode = new EventEmitter<string>();
+
+    /**
+     * Represents the boolean variable isLoading.
+     */
+    isLoading = input.required<boolean>();
+    #destroyRef = inject(DestroyRef);
+
+    ngAfterViewInit(): void {
+        this.listenInputChanges();
```

Comment:
@oidacra It is not necessary to start listening to the changes on `ngAfterViewInit` because the `FormControl` is able to start listening to these changes instantly in the `constructor`.

Reply:
deleted.